<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="taskType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Тип операції</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="addition">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Додавання</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="multiplication">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Множення</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="transposition">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Транспонування</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="matrixA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>321</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Матриця А</string>
    </property>
    <widget class="QTextBrowser" name="matrixAView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>301</width>
       <height>301</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="rowsA">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>340</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Число рядків:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Число стовпців:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="columnsA">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>380</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
    </widget>
    <widget class="QPushButton" name="generateA">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Згенерувати</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearA">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>420</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистити</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="matrixB">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>321</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Матриця B</string>
    </property>
    <widget class="QTextBrowser" name="matrixBView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>301</width>
       <height>301</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="rowsB">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>340</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Число рядків:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Число стовпців:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="columnsB">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>380</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
    </widget>
    <widget class="QPushButton" name="generateB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Згенерувати</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearB">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>420</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистити</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tickSettings">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Тактів на операцію</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Додавання:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Множення:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="ticksPerAddition">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>42</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="ticksPerMultiplication">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>42</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Присвоєння:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="ticksPerAssignment">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>42</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="taskPerformType">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Спосіб виконання</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="oneUnit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Один ПЕ</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="m1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Алгоритм М1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="m2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Алгоритм М2</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="matrixC">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>321</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Матриця C (результат)</string>
    </property>
    <widget class="QTextBrowser" name="matrixCView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>301</width>
       <height>301</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Число рядків:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Число стовпців:</string>
     </property>
    </widget>
    <widget class="QLabel" name="rowsC">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>340</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;невідомо&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="columnsC">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>380</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;невідомо&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearC">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>420</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистити</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Отримати</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
